src/test/scala/com/fulcrumgenomics/commons/io/IoTest.scala

```diff
@@ -216,4 +220,9 @@ class IoTest extends UnitSpec {
     val out = Io.readLines(f).toSeq
     out shouldBe in
   }
+  
+  "IoUtil.compressionLevel" should "be settable" in {
+    FakeIo.compressionLevel = 6
+    FakeIo.compressionLevel shouldEqual 6
```

Comment:
We use `shouldBe` in most places.  TIL: https://stackoverflow.com/questions/43923312/whats-the-difference-between-shouldbe-vs-shouldequal-in-scala